.github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md

```diff
@@ -0,0 +1,52 @@
+---
+name: 'Runtime Upgrade: [Upgrade Title/Version]'
+about: Checklist for tracking the progress of a runtime upgrade on the Paseo Testnet.
+title: ''
+labels: runtime-upgrade
+assignees: ''
+
+---
+
+# Paseo Testnet Runtime Upgrade Checklist
+
+## Preparation
+
+- [ ] **Upgrade Proposal Review**: Confirm that the upgrade proposal has been fully reviewed and approved.
+- [ ] **Impact Analysis Completed**: Assess and document the expected impact of the upgrade on existing systems and users.
+- [ ] **Communication Plan**: Draft and approve the communication plan for the upgrade.
+
+## Pre-Upgrade
+
+- [ ] **Schedule Upgrade Window**: Decide on and communicate the upgrade window to all stakeholders.
+- [ ] **Notify Stakeholders**: Distribute the communication plan through all relevant channels.
+- [ ] **Prepare Upgrade Guide**: Ensure detailed upgrade instructions are available and easy to follow.
+- [ ] **Validator Coordination**: Confirm that validators are informed and prepared for the upgrade.
+
+## Execution
+
+- [ ] **Final Pre-Upgrade Check**: Conduct a last-minute check to ensure all systems and stakeholders are ready.
+- [ ] **Deploy Upgrade**: Initiate the runtime upgrade process according to the planned procedure.
+- [ ] **Monitor Deployment**: Closely monitor the network for any immediate issues during the upgrade.
+
+## Validation
+
+- [ ] **Verify Network Stability**: Check that the network returns to normal operation post-upgrade.
+- [ ] **Confirm Upgrade Success**: Validate that the new runtime version is correctly implemented across the network.
+- [ ] **Stakeholder Feedback**: Collect and review feedback from validators, developers, and users.
+
+## Post-Upgrade
+
+- [ ] **Update Documentation**: Revise all relevant documentation to reflect changes introduced by the upgrade.
```

Comment:
We should revise everything beforehand and then update accordingly in post-upgrade.

Reply:
i agree with that, are you ok if i update the .md with that?